l3build.dtx

```diff
@@ -379,6 +379,8 @@
 % \item |--rerun| Run tests without unpacking/set up
 % \item |--show-log-on-error| To be used in addition to \texttt{--halt-on-error} and results
 %   in the full \texttt{.log} file of a failed test to be shown on the console
+% \item |--show-save-cmds| When tests fail, print the \texttt{l3build save} commands needed
+%   to regenerate the tests assuming that the failures were false negatives.
```

Comment:
When tests fail, print the \texttt{l3build save} commands needed%   to regenerate them after correction. Useful when writing tests.

Reply:
I wouldn't want to refer to corrections since the typical usecase is for changes to completely independent code which just happens to influence other tests. Anyway a separate documentation section has been added to expandon the usage.